tools/make/project/e2e.mk

```diff
@@ -0,0 +1,4 @@
+test-pw: ## Run E2E tests in a container
+	@docker-compose exec e2e sh -c "npm install -y --silent && npx playwright test $(filter-out $@,$(MAKECMDGOALS))"
```

Comment:
Omia huomiota taisteluista.

Mulla ei suoraan toiminut tämä test-pw komento, kaatu samoin tein Error 1 virheeseen. 

docker composen vanhemmat versiot yleensä on käyttänyt tota `docker-compose`, mutta v2 on alkanut käyttää `docker compose` tai ainakin suositellaan mielummin käytettäväksi
https://docs.docker.com/compose/migrate/#docker-compose-vs-docker-compose

![image](https://github.com/City-of-Helsinki/hel-fi-drupal-grants/assets/43133397/05570577-4dc6-4672-9d0f-ff3c56645d69)

Tän kun vaihtaa `@docker composer`, niin lähti pyörimään kontissa ok.  Varmaan vois muuttaa tähänkin, sama on käytössä myös docker.mk & robo.mk filuissa

Reply:
Joo, tää on nyt päivitetty 👍🏻